README.md

```diff
@@ -1,50 +1,49 @@
-# HQ - UI test
+# HQ - Frontend Coding Challenge
+## General Overview
+This challenge reflects a typical project you might encounter on our team. You will build a responsive marketplace where customers can add products to their cart, proceed to checkout, and successfully place an order. This test aims to assess your proficiency in frontend development, component creation, state management, API integration, user experience optimization, and more.
+
+### Setup Instructions
+1. **Environment Variables**
+    - Create a `.env` file in the root directory with the following:
+      ```
+      NEXT_PUBLIC_API_URL="http://localhost:8000"
+      ```
+
+2. **Backend Setup**
+    - Ensure your backend application is running as per the provided instructions.
+
+3. **Frontend Application**
+    - Start your frontend application by executing:
+      ```
+      yarn dev
+      ```
+    - Access your application at `http://localhost:3000`
+
+## Objectives
+  Clone the respository and complete the following tasks.
+
+### User Story
+  Customer is shown a responsive marketplace where they can add products to the cart. For this test, the customer will need to add more than one product to the cart before proceeding to checkout (figure out how this is going to work). In checkout, customers will be prompted for a delivery address on the first product that they can use for the second product. Each product in the cart will have a service fee that the customer can see as part of the overall total.
```

Comment:
Maybe rephrase it to "...In checkout, customers will be prompted for a delivery address. Inputted addresses should become delivery options for the other products." and "Each vendor charges a service fee that the customer can see as part of the overall total."